qemu/tests/cfg/driver_load_stress.cfg

```diff
@@ -233,3 +233,24 @@
             run_bgstress = virtio_mem_numa_basic
             requested-size_test_vmem0 = "8G 4G 0"
             driver_id_pattern = "(.*?):.*?VirtIO Viomem Driver"
+        - with_viosock:
+            no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+            no during_bg_test
```

Comment:
just curious, why not during_bg_test.

Reply:
This is not a valid test for viosock

Reply:
Hi @xiagao I will update the xml files and add test results later, thanks for your suggestions.